OCMock.podspec

```diff
@@ -18,6 +18,8 @@ Pod::Spec.new do |s|
   
   s.source       = { :git => "https://github.com/erikdoe/ocmock.git", :tag => "v3.1.3" }
   s.source_files  = "Source/OCMock/*.{h,m}"
+
+  s.platform     = :ios
```

Comment:
What does this have to look like if we want to build the pod for OS X and for iOS development?